test/python/algorithms/eigensolvers/test_vqd.py

```diff
@@ -107,9 +103,10 @@ def test_basic_operator(self, op):
             )
             np.testing.assert_array_almost_equal(job.result().values, result.eigenvalues, 6)
 
-    def test_full_spectrum(self):
+    @data(L_BFGS_B(), SPSA())
```

Comment:
If this takes a bit long to run, I think the other test you added is enough and we could remove this one 🙂